Bottom: b7b3726632c95160a455e9ae58f77cfdb6461db9
Top:    26e9149c3dc895caf3714619ecb1b06cb6088f54
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-31 10:18:50 -0700

Refresh of addrtype

---

diff --git a/man/ucmatose.1 b/man/ucmatose.1
index 73477ea..baf5837 100644
--- a/man/ucmatose.1
+++ b/man/ucmatose.1
@@ -4,10 +4,10 @@ ucmatose \- RDMA CM connection and simple ping-pong test.
 .SH SYNOPSIS
 .sp
 .nf
-\fIucmatose\fR [-s server_address] [-b bind_address] [-c connections]
-		[-C message_count] [-S message_size]
-\fIucmatose\fR -s server_address [-b bind_address] [-c connections]
-		[-C message_count] [-S message_size] [-t tos]
+\fIucmatose\fR [-s server_address] [-b bind_address] [-f address_format]
+		[-c connections] [-C message_count] [-S message_size]
+\fIucmatose\fR -s server_address [-b bind_address] [-f address_format]
+		[-c connections] [-C message_count] [-S message_size] [-t tos]
 .fi
 .SH "DESCRIPTION"
 Establishes a set of reliable RDMA connections between two nodes using the
@@ -22,6 +22,13 @@ This option must be specified by the client.
 \-b bind_address
 The local network address to bind to.
 .TP
+\-f address_format
+Specifies the format of the server and bind address.  Be default, the
+format is determined by getaddrinfo() as either being a hostname, an IPv4
+address, or an IPv6 address.  This option may be used to indicate that
+a specific address format has been provided.  Supported address_format
+values are: name, ip, ipv6, and gid.
+.TP
 \-c connections
 The number of connections to establish between the client and server.
 (default 1)
